guides/common/modules/proc_configuring-the-project-client-in-keycloak.adoc

```diff
@@ -0,0 +1,44 @@
+[id="configuring-the-project-client-in-keycloak_{context}"]
+= Configuring the {Project} Client in {Keycloak}
+
+Use this procedure to configure the {Project} client in the {Keycloak} web UI and create group and audience mappers for the {Project} client.
+
+.Procedure
+
+. In the {Keycloak} web UI, navigate to *Clients* and click the {Project} client.
+. Configure access type:
+* If you want your users to authenticate to {Project} using the web UI, from the *Access Type* list, select *confidential*.
+* If you want your users to authenticate to {Project} using the CLI, from the *Access Type* list, select *public*.
```

Comment:
I think we are mixing a lot with UI and CLI configuration with each that leads to confusion for customer. I would think current 6.8 documentation is alright and brings the isolation between web UI and CLI make more robust

Reply:
In the 6.8 documentation, there are many things that were not mentioned. It is a try to make the documentation more readable.

Can you mention specific changes that are of concern?

Reply:
After looking at it again come to realise that the common steps are aligned differently which is okay